### PR TITLE
hasura-cli: revert to previous version

### PR DESCRIPTION
**Pull request by dawidd6**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Looks like upstream project took down this release.
https://github.com/hasura/graphql-engine/releases/tag/v1.3.3
https://github.com/hasura/graphql-engine/releases/tag/v1.3.2

Issue found in: https://github.com/Homebrew/homebrew-core/pull/64679

### CODE REVIEW

File: Formula/hasura-cli.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   license "Apache-2.0"
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "f401fa465f27d50d1fcd467d53b2db03790f8248527514b49ca867679a622b5b" => :catalina
-    sha256 "62bf68a7d5118c8547849272425dcd49fa2be3847236cef3c3ca4e4233c972be" => :mojave
-    sha256 "1eae260ad4598bf74d835bf0923ef8323713e367d7389a4febb5134b7c821a33" => :high_sierra
+    sha256 "85e902904e462772a3f45e236178195a61a71daf53ca2dd488e8aafbd5374be7" => :catalina
+    sha256 "204859c85e1cbe73eeffb0a02641f23d607ead9aa4a90337c9c6e64a1b219c3e" => :mojave
+    sha256 "0b2eac67491041cf2ff84dc7a1b5668ee47c8aab366edc8fda3f943347f396ee" => :high_sierra
```
Comment (SMillerDev) [inline Formula/hasura-cli.rb:12]: This shouldn't change, but there should be a `version_scheme`

Comment (dawidd6) [reply Formula/hasura-cli.rb:12]: These are the checksums of previous bottles, so we don't have to upload them again.

What do you mean by `version_scheme`? Versioning of this project didn't change.

Comment (SMillerDev) [reply Formula/hasura-cli.rb:12]: > These are the checksums of previous bottles, so we don't have to upload them again.

I don't think brew is really equiped to deal with that scenario, or at least the audit isn't.

> What do you mean by version_scheme? Versioning of this project didn't change.

This is the only way to downgrade users in brew PRs

**Comment by chenrui333** [comment]
This is confusing when somebody created and deleted the tag. I'd vote to ignore the build failure. 

Adding `version_scheme` would be the last sort if all necessary.

**Comment by dawidd6** [comment]
Looks like they rereleased v1.3.3: https://github.com/hasura/graphql-engine/releases/tag/v1.3.3

Superseded by: https://github.com/Homebrew/homebrew-core/pull/65175

**Pull request closed without merging**